portafly/src/components/pages/accounts/listing/AccountsListingPage.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { useAsync } from 'react-async'
+import { useDocumentTitle, Loading, DataListProvider } from 'components'
+import { AccountsListingTable, generateColumns, generateRows } from 'components/pages/accounts/listing'
+import {
+  Alert,
+  PageSection,
+  TextContent,
+  Text,
+  Button,
+  Flex,
+  FlexItem,
+  FlexModifiers
+} from '@patternfly/react-core'
+import { getDeveloperAccounts } from 'dal/accounts'
+import { useTranslation } from 'i18n/useTranslation'
+import { PlusCircleIcon, ExportIcon } from '@patternfly/react-icons'
+import { IDeveloperAccount } from 'types'
+
+const AccountsListingPage: React.FunctionComponent = () => {
+  const { data: accounts, error, isPending } = useAsync(getDeveloperAccounts)
+  const { t } = useTranslation('audienceAccountsListing')
+  useDocumentTitle(t('title_page'))
+
+  const isMultitenant = false // TODO: get this somehow
+
+  const DataListTable = ({ accounts: _accounts }: { accounts: IDeveloperAccount[] }) => {
+    const columns = generateColumns(_accounts, t)
+    const rows = generateRows(_accounts, isMultitenant)
+
+    const initialState = {
+      table: { columns, rows }
+    }
+
+    return (
+      <DataListProvider initialState={initialState}>
+        <AccountsListingTable />
+      </DataListProvider>
+    )
+  }
```

Comment:
This could be extracted outside `AccountsListingPage`

Reply:
See my response to Damian's same comment